src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -1067,6 +1078,607 @@ TEST_F(MessageHelperTest,
   EXPECT_EQ(tested_enum, converted_enum);
 }
 
+TEST_F(MessageHelperTest,
+       SendUIChangeRegistrationRequestToHMI_AppNotValid_UNSUCCESS) {
+  MockApplicationSharedPtr app;
+  EXPECT_CALL(mock_application_manager, ManageHMICommand(_)).Times(0);
```

Comment:
@LevchenkoS  do you rely think that checking that SendUIChangeRegistrationRequestToHMI just send _something_ to hmi is enough?